File: level2/멀쩡한_사각형/solution.js
function solution(w, h) {
  var answer = w * h;
  var gcd = (min, max) => (max === 0 ? min : gcd(max, min % max));
  return answer - (w + h - gcd(w, h));
}

console.log(solution(8, 12));

File: level2/멀쩡한_사각형/Solution.java
import java.math.BigInteger;

class Solution {
    public long solution(int w, int h) {
        long answer = (long) w * (long) h;
        BigInteger gcd = BigInteger.valueOf(w).gcd(BigInteger.valueOf(h));

        return answer - ((long) w + (long) h - gcd.longValue());
    }

    public static void main(String[] args) {
        Solution s = new Solution();
        System.out.println(s.solution(8, 12));
    }
}
